winpr/libwinpr/sspi/Negotiate/negotiate.c

```diff
@@ -260,12 +180,12 @@ static const char* negotiate_mech_name(const sspi_gss_OID_desc* oid)
 		return "Unknown mechanism";
 }
 
-static const Mech* negotiate_GetMechByOID(sspi_gss_OID_desc oid)
+static const Mech* negotiate_GetMechByOID(WinPrAsn1_OID oid)
```

Comment:
Shall be changed to `static const Mech* negotiate_GetMechByOID(const WinPrAsn1_OID* oid)` and use a local variable later